src/lib/resource/interceptor/token-refresher.es

```diff
@@ -54,8 +43,8 @@ class TokenRefresher {
   **/
   constructor (opts) {
     this.resetAttempts()
-    if (tokenRefresherSingletonInstance) {
-      return tokenRefresherSingletonInstance
+    if (this.tokenRefresherSingletonInstance) {
```

Comment:
I did it because of the UT's, I need to delete the instance aftearEach test. It seems to be working as always. An alternative could be to export the instance variable I had before, what do you prefer?